<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IWIViewer</class>
 <widget class="QWidget" name="IWIViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>909</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <widget class="QLabel" name="label_Title">
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>IWI File 0</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_17">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Header</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Magic:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_Magic">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>9</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_26">
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Version:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_Version">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="suffix">
             <string> B</string>
            </property>
            <property name="maximum">
             <number>1000000000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_14">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Info</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_40">
            <item>
             <widget class="QLabel" name="label_38">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Depth:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_Depth">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="maximum">
               <number>1000000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_41">
            <item>
             <widget class="QLabel" name="label_35">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Format</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Format">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>9</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_43">
            <item>
             <widget class="QLabel" name="label_39">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Height:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_Height">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="maximum">
               <number>1000000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_44">
            <item>
             <widget class="QLabel" name="label_40">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Width:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_Width">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="maximum">
               <number>1000000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_45">
            <item>
             <widget class="QLabel" name="label_41">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Usage</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_Usage">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="maximum">
               <number>1000000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_21">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Mipmaps</string>
         </property>
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_25">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_32">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Select Mipmap:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_Mipmap">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>9</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_55">
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Size: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_MipmapSize">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="maximum">
             <number>1000000000</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_56">
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Offset:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinBox_MipmapOffset">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="maximum">
             <number>1000000000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_26">
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Preview</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_22">
        <item>
         <spacer name="verticalSpacer_11">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_Image">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>600</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>600</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid white;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_12">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_13">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>116</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
